src/CMakeLists.txt

```diff
@@ -152,53 +153,22 @@ endif()
 option(BUILD_UI "Build antares ui simulator" ON)
 message(STATUS "Build antares ui simulator: ${BUILD_UI}")
 
-option(BUILD_DEPS "Build all dependencies" OFF)
-message(STATUS "Build all dependencies: ${BUILD_DEPS}")
-
-option(USE_ORTOOLS_STABLE "Use OR-Tools stable instead of RTE OR-Tools branch with Sirius support" OFF)
-message(STATUS "Use OR-Tools stable instead of RTE OR-Tools branch with Sirius support: ${USE_ORTOOLS_STABLE}")
-
-# IF BUILD_DEPS=ON THEN Force all BUILD_*=ON
-include(CMakeDependentOption)
-
-CMAKE_DEPENDENT_OPTION(BUILD_sirius "Build sirius solver dependency Library" OFF
-		"NOT BUILD_DEPS" ON)
-message(STATUS "Build sirius solver: ${BUILD_sirius}")
-
-CMAKE_DEPENDENT_OPTION(BUILD_ortools "Build BUILD_ortools dependency Library" OFF
-		"NOT BUILD_DEPS" ON)
-message(STATUS "Build OR-Tools: ${BUILD_ortools}")
-
-CMAKE_DEPENDENT_OPTION(BUILD_wxWidgets "Build wxWidgets dependency Library" OFF
-  "NOT BUILD_DEPS" ON)
-message(STATUS "Build wxWidgets: ${BUILD_wxWidgets}")
-
-CMAKE_DEPENDENT_OPTION(BUILD_OPENSSL "Build OpenSSL dependency Library" OFF
-  "NOT BUILD_DEPS" ON)
-message(STATUS "Build BUILD_OPENSSL: ${BUILD_OPENSSL}")
-
-CMAKE_DEPENDENT_OPTION(BUILD_curl "Build curl dependency Library" OFF
-  "NOT BUILD_DEPS" ON)
-message(STATUS "Build BUILD_curl: ${BUILD_curl}")
-
-CMAKE_DEPENDENT_OPTION(BUILD_zlib "Build zlib dependency Library" OFF
-  "NOT BUILD_DEPS" ON)
-message(STATUS "Build BUILD_zlib: ${BUILD_zlib}")
-
 #Define install directory
 if (NOT DEPS_INSTALL_DIR)
     SET(DEPS_INSTALL_DIR ${CMAKE_CURRENT_SOURCE_DIR}/../../rte-antares-deps-${CMAKE_BUILD_TYPE})
+
+else()
+    if(NOT IS_ABSOLUTE ${DEPS_INSTALL_DIR})
+        SET(DEPS_INSTALL_DIR ${CMAKE_CURRENT_SOURCE_DIR}/../${DEPS_INSTALL_DIR})
```

Comment:
Need to use `${CMAKE_CURRENT_SOURCE_DIR}/../${DEPS_INSTALL_DIR}` with `.. `because CMakeList.txt is not in checkout directory but in src directory

Reply:
Yes there is an issue with the relative path transformation. We should also use `../..`.

To be changed.

Reply:
GitHub action CI show an error with the change.

In case DEPS_INSTALL_DIR is relative we don't have to use `../..`. If we use this, the path is not relative to the checkout directory but with the directory above.